src/main/java/org/candlepin/resource/OwnerContentResource.java

```diff
@@ -72,12 +73,41 @@ public OwnerContentResource(OwnerCurator ownerCurator,
         this.idGenerator = idGenerator;
     }
 
+    /**
+     * Validates that fields that are required for content creation are populated and non-empty.
+     *
+     * @param dto
+     *  the content DTO to validate
+     *
+     * @throws BadRequestException
+     *  if the DTO does not pass validation
+     */
+    private void validateContentForCreation(ContentDTO dto) {
+        this.validator.validateCollectionElementsNotNull(dto::getModifiedProductIds);
+
+        if (dto.getLabel() == null || dto.getLabel().isEmpty()) {
+            throw new BadRequestException("content label cannot be null or empty");
```

Comment:
These strings should probably be translated

Reply:
Yup. Fixed.